app/code/Magento/Catalog/Controller/Adminhtml/Product.php

```diff
@@ -10,6 +10,7 @@
 /**
  * Catalog product controller
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @api
```

Comment:
Lets move the `@SuppressWarnings` at the end, and `@api` should follow class description.